opentelemetry-api/src/common.rs

```diff
@@ -464,7 +464,7 @@ pub struct InstrumentationLibrary {
     pub schema_url: Option<Cow<'static, str>>,
 
     /// Specifies the instrumentation scope attributes to associate with emitted telemetry.
-    pub attributes: Option<Vec<KeyValue>>,
```

Comment:
I removed `Option` here as I don't see the point in having both states (`None` and `Some(vec![])`) represent the logical case of having no attributes.